source/install.rst

```diff
@@ -1 +1,297 @@
-1. Setting up your environment
+############
+Installation
+############
+
+Before we start, we need to prepare our environment. Below are some instructions on installing the latest version of Python. Here we install version 3.6.5.
+
+Windows
+=======
+
+You can download Python for Windows directly from `python.org`_. Look for version 3 and above.
+After downloading the file  ``*.msi``, open it and follow the instructions.
+It is important to remember the path of installation, i.e. the directory, as we will need this information during the :ref:`installation of tools <tools>`.
+
+.. note::
+    Remember to check "Add Python 3.6 to PATH" when running the installer . This way you will be able to run Python form
+    console by just writing ``python``.
+
+
+Linux (Ubuntu, Fedora, etc.) or Mac
+===================================
+
+In order to check our version of Python, enter the following in the command line:
+
+.. code-block:: sh
+
+    $ python --version
+    Python 3.6.5
+
+If the ``python`` command is not available or the wrong version appears try using ``python3`` command:
+
+Ubuntu
+------
+
+In the command line enter::
+
+    sudo apt-get install python3.6
+
+Fedora
+------
+
+In the command line enter::
+
+    sudo yum install python3.6
+
+OS X
+----
+
+Download and install the package for your system version from `python.org`_ .
+
+
+Other
+-----
+
+Use the system of packages adequate for your distribution. If there is no adequate system
+or you cannot find python, you can install it using the sources on the `python.org`_. website.
+A compiler and readline library will be required.
+
+
+
+.. _tools:
+
+Tools
+=====
+
+Windows command line
+--------------------
+
+We will do most of our work on the command line. To activate the command line in Windows,
+press ``Win+R``. In the open window, type ``cmd`` and click ``OK``.
+A new window will appear with white text on a black background:
+
+.. code-block:: bat
+
+    Microsoft Windows [Version 6.1.7601]
+    Copyright (c) 2009 Microsoft Corporation. All rights reserved.
+
+
+    C:\Users\Name>
+
+Text may be different depending on the version of Windows you use.
+
+``C:\Users\Name>``  is a prompt. It informs us of the directory (or folder) in which we currently are and waits for a command.
+Later during the workshop ``C:\Users\Name>`` we will refer to it with ``~$``, independently of your
+operating system (Windows, Linux, MacOS).
+
+Using the command line, you can move around the contents of the disc (in the same way as in
+``My Computer``).  You can do that by entering commands and pressing ``Enter``.
+Use the following commands:
+
+``dir``
+    Displays the contents of the current directory. For example, if the ``prompt``
+    shows  ``C:\Users\Name``, the ``dir`` command displays the contents of our home directory.
+
+``cd directory``
```

Comment:
I would add that "cd" stands for "change directory", to make it easier to understand and remember. And "mkdir" for "make directory".